<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LockFileWaiter</class>
 <widget class="QDialog" name="LockFileWaiter">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Locked files have been found</string>
  </property>
  <property name="windowIcon">
   <iconset resource="libqpacman.qrc">
    <normaloff>:/pics/emblem-locked.svg</normaloff>:/pics/emblem-locked.svg</iconset>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;The following files are locked:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="filesList"/>
   </item>
   <item>
    <widget class="QLabel" name="infoLabel">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;Either release or delete (if nothing is really working now)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>Delete</string>
       </property>
       <property name="icon">
        <iconset resource="libqpacman.qrc">
         <normaloff>:/pics/edit-delete.svg</normaloff>:/pics/edit-delete.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="libqpacman.qrc">
         <normaloff>:/pics/dialog-cancel.svg</normaloff>:/pics/dialog-cancel.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quitButton">
       <property name="text">
        <string>Quit</string>
       </property>
       <property name="icon">
        <iconset resource="libqpacman.qrc">
         <normaloff>:/pics/application-exit.svg</normaloff>:/pics/application-exit.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="libqpacman.qrc"/>
 </resources>
 <connections/>
</ui>
